src/main/java/com/transportcompany/transportcompany/services/UserService.java

```diff
@@ -5,9 +5,13 @@
 import java.util.List;
 
 public interface UserService {
+    UserDTO createUser(UserDTO userDTO);
```

Comment:
Please leave an empty row between the class name and the first line of code. Also, please don't forget about code formatting.

Reply:
Done